src/main/groovy/io/openliberty/tools/gradle/tasks/InstallFeatureTask.groovy

```diff
@@ -35,6 +36,15 @@ class InstallFeatureTask extends AbstractServerTask {
         })
     }
 
+    // DevMode uses this option to provide the location of the
+    // temporary serverDir it uses after a change to the server.xml
+    private String serverDirectoryParam;
+
+    @Option(option = 'serverDir', description = '(Optional) Server Directory to get the list of features from.')
```

Comment:
Change description to `(Optional) Server directory containing the configuration files to get the list of features from.`

Reply:
Fixed